src/components/dataset/show.vue

```diff
@@ -84,7 +83,7 @@ export default {
     const { tabPath, tabClass } = useTabs(datasetPath());
     return {
       project, dataset, ...resourceStates([project, dataset]),
-      projectPath, datasetPath, tabPath, tabClass, canRoute
```

Comment:
I searched for all uses of `datasetPath()` to see where to add `DatasetLink`. I noticed that this component doesn't use `datasetPath()` in the template, so it doesn't need to return it from `setup()`.